src/managers/PermissionManager.js

```diff
@@ -69,13 +70,12 @@ class PermissionManager {
         'administrator',
       );
 
-      const token = await this._generateAdminToken();
+      const aToken = await this._generateAdminToken();
```

Comment:
Sorry, i've found this half an hour before. Already implemented in the changings to spark-protocol :(